### PR TITLE
update usage limitation of partitioned-raft-kv

### PR DESCRIPTION
**Pull request by BusyJay**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

...for 7.1.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: partitioned-raft-kv.md [hunk 1]
```diff
@@ -35,7 +35,6 @@ v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例
 
 由于该功能为实验特性，目前有以下限制：
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:36]: 在 7.1 分区 Raft KV 还是实验特性吗？

File: partitioned-raft-kv.md [hunk 2]
```diff
@@ -35,7 +35,6 @@ v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例
...
 
-* 暂不支持 TiDB Lightning、TiCDC、BR、Dumping 等数据导入、同步和备份工具。
 * 暂不支持 tikv-ctl 命令行管理工具。
-* 不支持同时和 TiFlash 使用。
-* 需要在创建集群时启用，不支持在集群创建后开启。
+* 初始化以后不支持开启或者关闭。
+* 暂不支持 titan。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:40]:
```suggestion
* 暂不支持 Titan。
```

**Comment by Oreoxmt** [comment]
@tonyxuqqi @benmaoer PTAL

File: partitioned-raft-kv.md [hunk 3]
```diff
@@ -35,7 +35,8 @@ v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例
 
 由于该功能为实验特性，目前有以下限制：
 
-* 暂不支持 TiDB Lightning、TiCDC、BR、Dumping 等数据导入、同步和备份工具。
+* 暂不支持 基于 EBS 的 snapshot 备份。
+* 暂不支持 flashback、Online unsafe recovery、Titan。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:39]:
```suggestion
* 暂不支持基于 EBS 的快照备份。
* 暂不支持 `FLASHBACK` 语句、Online Unsafe Recovery 和 Titan。
```

**Review by hfxsd — APPROVED** [review]

File: partitioned-raft-kv.md [hunk 4]
```diff
@@ -35,7 +35,8 @@ v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例
 
 由于该功能为实验特性，目前有以下限制：
 
-* 暂不支持 TiDB Lightning、TiCDC、BR、Dumping 等数据导入、同步和备份工具。
+* 暂不支持基于 EBS 的快照备份。
+* 暂不支持 `FLASHBACK` 语句、Online Unsafe Recovery 和 Titan。
```
Comment (tonyxuqqi) [inline partitioned-raft-kv.md:39]: FLASHBACK 7.1是支持的

Comment (Oreoxmt) [inline partitioned-raft-kv.md:39]:
```suggestion
* 暂不支持 Online Unsafe Recovery 和 Titan。
```

Comment (Oreoxmt) [reply partitioned-raft-kv.md]: update in https://github.com/pingcap/docs-cn/pull/13688/commits/266bd3708d0f52ad3d1195b2efd55d6c3b03c83b

File: partitioned-raft-kv.md [hunk 5]
```diff
@@ -35,7 +35,8 @@ v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例
 
 由于该功能为实验特性，目前有以下限制：
 
-* 暂不支持 TiDB Lightning、TiCDC、BR、Dumping 等数据导入、同步和备份工具。
+* 暂不支持基于 EBS 的快照备份。
+* 暂不支持 Online Unsafe Recovery 和 Titan。
 * 暂不支持 tikv-ctl 命令行管理工具。
-* 不支持同时和 TiFlash 使用。
-* 需要在创建集群时启用，不支持在集群创建后开启。
+* 暂不支持同时和 TiFlash 使用。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:41]:
```suggestion
```
@tonyxuqqi PTAL

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**